master/buildbot/steps/master.py

```diff
@@ -213,6 +213,22 @@ def run(self):
         return defer.succeed(SUCCESS)
 
 
+class Assert(BuildStep):
+    name = 'Assert'
+    description = ['Checking..']
+    descriptionDone = ['Checked']
+    renderables = ['check']
+
+    def __init__(self, check, **kwargs):
+        BuildStep.__init__(self, **kwargs)
+        self.check = check
+
+    def run(self):
+        if self.check:
+            return defer.succeed(SUCCESS)
```

Comment:
Is that useful for debugging: `'checked renderer(<function test_renderer at 0x7fd8d139c230>)'` ?